src/core/lender/distributor/distributor.sol

```diff
@@ -44,36 +37,42 @@ contract Distributor is DSNote {
     function deny(address usr) public auth note { wards[usr] = 0; }
     modifier auth { require(wards[msg.sender] == 1); _; }
 
-    ManagerLike public manager;
+    address public shelf;
```

Comment:
```suggestion
    ShelfLike public shelf;
```

Reply:
Currently we need `shelf` as address more often. 

We don't want to store both. I want to check what is cheaper `ShelfLike(...)` or `address(...)`

Reply:
technically it is in both cases anyway an address